libs/dev-dock/frontend/src/dev_dock.tsx

```diff
@@ -557,16 +570,16 @@ const Container = styled.div`
   /* Slide the dock up when closed */
   &.closed {
     /* Slide up enough to hide the shadow */
-    top: -300px;
-    transition: top 0.15s ease-out;
+    transform: translateY(-100%);
+    transition: all 0.15s ease-out;
     /* Move the handle down a bit to compensate for sliding up extra to hide the
      * shadow */
     #handle {
-      top: 20px;
+      top: 60px;
     }
   }
   /* Slide the dock down when open */
-  transition: top 0.15s ease-out;
+  transition: all 0.15s ease-out;
```

Comment:
Now the height of the dev dock is variable, so we couldn't use translations based on fixed px values.